src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Default {
```

Comment:
The class `Circle` should not extend `Default`. Consider using an interface to define the behavior instead of using an abstract class.